war/src/main/java/io/jenkins/pluginhealth/scoring/config/SecurityConfiguration.java

```diff
@@ -40,7 +40,13 @@ public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Excepti
             .authorizeHttpRequests(request ->
                 request
                     .requestMatchers(HttpMethod.GET, "/js/*", "/style.css", "/svg/*").permitAll()
-                    .requestMatchers(HttpMethod.GET, "/api/scores", "/", "/probes", "/probes/*", "/scores", "/scores/*").permitAll()
+                    .requestMatchers(HttpMethod.GET,
+                        "/api/scores",
+                        "/",
+                        "/probes", "/probes/*",
+                        "/scores", "/scores/*",
+                        "/actuator/*"
```

Comment:
Because of https://github.com/jenkins-infra/helm-charts/blob/bbcf80f50f10fa294024d975eb08a54acb251bb0/charts/plugin-health-scoring/values.yaml#L53-L54